### ISSUE TITLE
Helm Push with username and password

### ISSUE THREAD

**Issue by Jean-Jacques-Nuoan** [manusa/helm-java#122]
how to push with username and password ,like  'helm cm-push'

**Comment by manusa** [comment]
That's a push implemented by a plugin.

Plugin management and usage hasn't been implemented yet.

- https://github.com/chartmuseum/helm-push

**Comment by Jean-Jacques-Nuoan** [comment]
> That's a push implemented by a plugin.
> 
> Plugin management and usage hasn't been implemented yet.
> 
> * https://github.com/chartmuseum/helm-push

Do we currently have plans to implement this part

**Comment by manusa** [comment]
> Do we currently have plans to implement this part

Yes, the goal of helm-java is to have feature parity with the Helm CLI.

However, I can't say when this specific feature will be implemented.
